test/e2e/tests/test_authpolicy_generation_stability.py

```
@@ -0,0 +1,149 @@
+    _wait_for_gateway_auth_enforced,
+)
+
+log = logging.getLogger(__name__)
+
+STABILITY_WINDOW = int(os.environ.get("E2E_GENERATION_STABILITY_WINDOW", "60"))
+POLL_INTERVAL = 5
+
+
+def _get_authpolicy_generation(name: str = GATEWAY_AUTH_POLICY_NAME, namespace: str = GATEWAY_NAMESPACE) -> int | None:
+    """Return metadata.generation of the AuthPolicy, or None if not found."""
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)